README.md

```diff
@@ -26,7 +26,6 @@ Currently, this list of dependencies is as follows:
 - libyang-cpp_1.0.73_amd64.deb
 - python3-yang_1.0.73_amd64.deb
 - redis_dump_load-1.1-py3-none-any.whl
-- swsssdk-2.0.1-py3-none-any.whl
```

Comment:
>swsssdk [](http://example.com/codeflow?start=2&length=7)

Add "python-swsscommon_1.0.0_amd64.deb" into the list?

Reply:
Fixed.